### PR TITLE
Test for converting checkbox/multiple answers

### PR DESCRIPTION
**Pull request by benjefferies**
### What is the context of this PR?
There was no test covering multiple answers for the answer converter

### How to review 
Tests should pass

### CODE REVIEW

File: tests/app/submitter/test_converter.py [hunk 1]
```diff
@@ -204,6 +204,25 @@ def test_convert_census_answers(self):
             self.assertEqual(answer_object['data'][3]['answer_instance'], 0)
             self.assertEqual(answer_object['data'][3]['value'], '63 Somewhere')
 
+    def test_convert_census_answers_multiple_answers(self):
+        with self.application.test_request_context():
+            routing_path = [location(group_id='personal details', block_id='about you')]
+            answers = [create_answer('name', ['Joe Bloggs', 'Fred Bloggs'], group_id='personal details', block_id='about you')]
```
Comment (jonnyshaw89) [inline tests/app/submitter/test_converter.py]: Would multiple names not be 2 different answer instances?

Comment (benjefferies) [reply tests/app/submitter/test_converter.py]: Not if you have a checkbox answer with a selection of names ;). I will make the answer look more checkboxy

Comment (benjefferies) [reply tests/app/submitter/test_converter.py]: Done

**Review by jonnyshaw89 — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Comment by jonnyshaw89** [comment]
👍

**Pull request merged**